index.html

```diff
@@ -109,101 +109,32 @@ <h2>Our Roles</h2>
               <h3>Programmers</h3>
               <p>
                 If you're interested in programming, we're looking for
-                programmers who have worked with: web development (html, css,
-                js), data science (python, r), and mobile development (swift,
-                java, kotlin, react native)
+                people with experience in web development (HTML, CSS, and 
+                JavaScript), data science (Python, R), and mobile development 
+                (Swift, Java, Kotlin, and React Native).
               </p>
             </li>
             <li>
               <span class="icon major style3 fa-copy"></span>
               <h3>Design & Copy</h3>
               <p>
-                Designers & Copy Writers are responsible for creating website
-                designs (using Figma, Canva, etc) and writing the content that
-                appear on websites.
+                Designers and copywriters are responsible for creating website
+                designs (using tools such as Figma and Canva), and writing 
+                content that appears on websites.
               </p>
             </li>
             <li>
               <span class="icon solid major style5 fa-cookie"></span>
               <h3>Marketing & Outreach</h3>
               <p>
                 Members on the outreach and marketing teams create graphics for
-                Facebook and Instagram to promote our products and recruit new
-                members. Additionally, they are responsible for reaching out to
+                social media to promote our products and recruit new members. 
+                Additionally, they are responsible for reaching out to
                 nonprofits to find new projects.
               </p>
             </li>
           </ul>
         </section>
-

```

Comment:
Would it be possible to keep this section just for the sake of in case we need the code later?

Reply:
Sure! I don't know how to revert it. But, here is a link to it: https://github.com/code4charity-ncssm/code4charity-ncssm.github.io/blob/main/index.html#L139-L206

Reply:
wait I can just add it back oops

Reply:
it should be done now